File: backend/src/main/java/me/performancereservation/domain/admin/repository/AdminManagerRequestRepository.java
package me.performancereservation.domain.admin.repository;

import me.performancereservation.domain.user.entity.ManagerRequest;
import me.performancereservation.domain.user.enums.ManagerRequestStatus;
import org.springframework.data.domain.Page;
import org.springframework.data.domain.Pageable;
import org.springframework.data.jpa.repository.JpaRepository;

public interface AdminManagerRequestRepository extends JpaRepository<ManagerRequest, Long> {
    Page<ManagerRequest> findAllByStatusOrderByCreatedAt(ManagerRequestStatus status, Pageable pageable);
}

File: backend/src/main/java/me/performancereservation/domain/admin/repository/AdminManagerRequestRepository.kt
package me.performancereservation.domain.admin.repository

import me.performancereservation.domain.user.entitiy.ManagerRequest
import me.performancereservation.domain.user.enums.ManagerRequestStatus
import org.springframework.data.domain.Page
import org.springframework.data.domain.Pageable
import org.springframework.data.jpa.repository.JpaRepository

interface AdminManagerRequestRepository : JpaRepository<ManagerRequest, Long> {
    fun findAllByStatusOrderByCreatedAt(status: ManagerRequestStatus, pageable: Pageable): Page<ManagerRequest>
}
